modules/platform/nuxeo-target-platforms/nuxeo-target-platforms-core/src/main/java/org/nuxeo/targetplatforms/core/descriptors/TargetDescriptor.java

```diff
@@ -39,13 +41,15 @@
 @SuppressWarnings("deprecation")
 public class TargetDescriptor {
 
-    private static final Log log = LogFactory.getLog(TargetDescriptor.class);
+    private static final Logger log = LogManager.getLogger(TargetDescriptor.class);
 
     @XNode("@id")
+    @XRegistryId
     String id;
 
-    @XNode("@enabled")
-    Boolean enabled;
+    @XNode(value = XEnable.ENABLE, fallback = "@enabled", defaultAssignment = "true")
+    // registry needs to list enabled descriptors
```

Comment:
Clarify the comment: `// no @XEnable because the registry needs to know which descriptors are enabled or not`